### ISSUE TITLE
CESM2.1.0 qualifiers too specific

### ISSUE THREAD

**Issue by gold2718** [ESCOMP/cesm#111]
### Summary of Issue:
In CESM2.1.0, some refcase qualifiers (e.g., for `RUN_REFCASE`) were introduced which are possibly too specific about the compset match. While these items are currently working, they will not work with a proposed CIME modification (see ESMCI/cime#3055).
### Expected behavior and actual behavior:
Problem: entries assume no components after WW3 (or any WAV component).
Expect: more general patterns
### What is the changeset ID of the code, and the machine you are using:
cesm2.1.0
### have you modified the code? If so, it must be committed and available for testing:
no
### Suggested change
I think we can change these (24) entries from:
```
compset="1850_CAM60_CLM50%BGC-CROP.*_CICE.*_POP2%ECO.*_MOSART_CISM2%NOEVOLVE_WW3_BGC%BDRD"
```
to
```
compset="1850_CAM60_CLM50%BGC-CROP.*_CICE.*_POP2%ECO.*_MOSART_CISM2%NOEVOLVE_WW3.*_BGC%BDRD"
```
I ran a test with this substitution with and without my proposed CIME change (which added `_SESP` after `WW3`). I think making this simple change will allow us to then update CIME to a version with the new compset scheme.
Thoughts? @klindsay28? @jedwards4b? @billsacks? @mvertens?

**Comment by mvertens** [comment]
I think this is a good way to proceed and will bring in the functionality you proposed today - which I think is important to bring in as soon as we can.

**Comment by billsacks** [comment]
I completely agree.

**Comment by billsacks** [comment]
Actually: If it's just the refcases that have this problem, then I think we don't need to do anything. I agree that we need to make matches more general moving forward, but there are no refcases in master (i.e., the cesm2.2 branch), and presumably we would only make this cime change on master, not on maint-5.6?

**Comment by jedwards4b** [comment]
I agree with Bill - this change is targeted to cesm2.2 which has no ref cases.

**Comment by klindsay28** [comment]
I can see that this is not an issue within the context of https://github.com/ESMCI/cime/pull/3055, i.e., adding a new stub IAC. However,
@gold2718, at https://github.com/ESMCI/cime/pull/3055#issuecomment-479683043, mentions

> If stub components are inserted into the compset, the search pattern will fail because there will be one or more stub components between WW3 and BGC%BDRD.
This is not happening now due to a flaw in the current logic for adding SESP (same counting problem that is in the @shippert code above).

I think that flaw of not inserting SESP into compset longnames is present on maint-5.6. If that flaw is fixed on maint-5.6, then I think that this issue becomes relevant. I think this issue can only be set aside if it is decided to not fix the flaw mentioned by @gold2718 on maint-5.6.

**Comment by gold2718** [comment]
@klindsay28, This is what @billsacks and @jedwards4b are saying. We will only be making this change on master so no changes are planned on maint-5.6. As far as we know, the current code will not cause any problems in CESM2.1 so the stub component fix will only go into CESM2.2.x and future releases.

Therefore, I think I can close this issue since no actions are required (future compset RE matches are scrutinized to make sure they allow for stub components).
